### DISCUSSION TITLE
v2 dev server with grunge stack / arc

### DISCUSSION THREAD

**Discussion by Xiphe** [remix-run/remix#6700] [Q&A] [answered]
Wondering if there are any plans / concepts you could share concerning the architect based grunge stack and if you have any plans or see a way to have this setup opt into HRM/HDR?

**Comment by meza** [comment]
I'd love an update on this too.

**Comment by MichaelDeBoey** [comment, marked as answer]
https://github.com/remix-run/grunge-stack/pull/154 will implement `v2_dev` behavior
